common-rest/src/main/java/org/uniprot/api/rest/validation/error/ResponseExceptionHandler.java

```diff
@@ -1,25 +1,5 @@
 package org.uniprot.api.rest.validation.error;
 
-import static java.util.Collections.emptyList;
-import static java.util.Collections.singletonList;
-import static org.uniprot.api.rest.output.UniProtMediaType.DEFAULT_MEDIA_TYPE_VALUE;
-import static org.uniprot.api.rest.output.UniProtMediaType.UNKNOWN_MEDIA_TYPE;
-import static org.uniprot.api.rest.request.HttpServletRequestContentTypeMutator.ERROR_MESSAGE_ATTRIBUTE;
-import static org.uniprot.api.rest.validation.error.ResponseExceptionHelper.addDebugError;
-import static org.uniprot.api.rest.validation.error.ResponseExceptionHelper.getBadRequestResponseEntity;
-import static org.uniprot.api.rest.validation.error.ResponseExceptionHelper.getContentTypeFromRequest;
-import static org.uniprot.core.util.Utils.nullOrEmpty;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Locale;
-import java.util.Objects;
-import java.util.stream.Collectors;
-
-import javax.servlet.http.HttpServletRequest;
-import javax.validation.ConstraintViolation;
-import javax.validation.ConstraintViolationException;
-
 import org.apache.catalina.connector.ClientAbortException;
```

Comment:
General comment, please be consistent with Statistics and Statistic. Use Statistics

Reply:
done